shopinvader/security/shopinvader_security.xml

```diff
@@ -6,6 +6,10 @@
         <field name="sequence">30</field>
     </record>
 
+    <record id="group_shopinvader_partner_binding" model="res.groups">
+        <field name="name">Shopinvader partner binding</field>
+    </record>
+
     <record id="group_shopinvader_manager" model="res.groups">
         <field name="name">Shopinvader Manager</field>
```

Comment:
```suggestion
        <field name="name">Manager</field>
```

While you're at it, could you please rename this group?
The application name is redundant here, as it's already taken from the `ir.module.category`. [See for example](https://github.com/odoo/odoo/blob/444b00a47a49f334037e65a244fe5a51e49ee28e/addons/purchase/security/purchase_security.xml#L17)

Reply:
Done. Thanks for the review